src/haz3lweb/LangDocMessages.re

```diff
@@ -164,6 +165,27 @@ let triv_exp: form = {
   };
 };
 
+let deferral_exp_group = "deferral_exp_group";
+let deferral_exp: form = {
+  let explanation = {
+    message: "Deferral expression. This marks a placeholder for some part of the argument expression in a partial application whose value supply is deferred to future applications.",
```

Comment:
"Deferral expression. This marks an argument that has not yet been applied in a partial application."